src/app/src/main/java/io/github/lonamiwebs/stringlate/utilities/ContextUtils.java

```diff
@@ -25,6 +25,6 @@ public boolean isConnectedToInternet(@Nullable @StringRes Integer warnMessageStr
         if (!result && warnMessageStringRes != null)
             Toast.makeText(_context, _context.getString(warnMessageStringRes), Toast.LENGTH_SHORT).show();
 
-        return result;
+        return !result;
```

Comment:
Considering this function now does literally the opposite, shouldn't we rename it as well? Maybe we can rename it to `isOffline` directly. Much shorter.

Reply:
I feel like I should probably leave that to you :/